ansible-ipi-install/roles/installer/templates/install-config-virtualmedia.j2

```diff
@@ -122,7 +122,7 @@ platform:
 {% endif %}
 {% endfor %}
 {% endif %}
-pullSecret: '{{ pullsecret }}'
+pullSecret: '{{ pullsecret | to_json }}'
```

Comment:
@smalleni Instead of this can we format this variable in `set-deployment-tasks` role.
```
- set_fact:
    pullsecret: "{{ pullsecret | to_json }}"
```